docs/protocol.md

```diff
@@ -590,6 +591,10 @@ Value is string "host:port".
 
 Send claim message to `host:port` when response is being sent.
 
+### Transport Header `cn` -- Caller Name
+
+Value is name of the service making the call.
```

Comment:
Having the service name makes sense, my question is whether it needs to be sent with every request vs sent once during initialization handshake.